metaflow/plugins/aws/aws_utils.py

```diff
@@ -101,4 +102,17 @@ def compute_resource_attributes(decos, compute_deco, resource_defaults):
         if compute_deco.attributes.get(k) is not None:
             result[k] = str(compute_deco.attributes[k] or "0")
 
+    if "cpu" in result and MAX_CPU_PER_TASK:
```

Comment:
nit: I would reverse the check order to benefit from lazy evaluation (checking a bool is much faster).